doc/bundler/development/DEBUGGING.md

```diff
@@ -102,4 +117,4 @@ source "file:///[path to repo's bundler directory]/tmp/1/gems/remote1/"
 gem "rack", '=0.9.1'
 ```
 
-Then you can test in the same way as you did in the previous example when fetching the gem from the RubyGems source.
\ No newline at end of file
+Then you can test in the same way as you did in the previous example when fetching the gem from the RubyGems source.
```

Comment:
Strange diff here. I guess your editor added a missing final new line? No big deal, just got curious about the confusing diff!